terraform/modules/cloudfoundry/outputs.tf

```
@@ -105,6 +105,10 @@ output "logsearch_archive_bucket_name" {
   value = module.logsearch-archive.bucket_name
 }
 
+output "logs-opensearch_archive_bucket_name" {
```

```suggestion
output "logs_opensearch_archive_bucket_name" {
```

Can we use underscores in the output names instead of hyphens for consistency?